### ISSUE TITLE
Add default action/effect

### ISSUE THREAD

**Issue by zoulja** [greenshot/greenshot#240]
In 99.999% of cases I apply 'Drop shadow' to my screenshot for better readability and then send it.
It would be really great if Greenshot could automate it and save me 5 mouse clicks

**Comment by kirkilj** [comment]
+1

Same percentage for me. If undo can revert the default effect for the 1% of cases, all the better.

**Comment by Lakritzator** [comment]
I'd love to see someone describe a proper use case. Like how would one configure this, does it depend on the target? What when the user **doesn't** want the effect? What when one wants to change the effect setting? Etc...

**Comment by zoulja** [comment]
> I'd love to see someone describe a proper use case. Like how would one configure this, does it depend on the target? What when the user **doesn't** want the effect? What when one wants to change the effect setting? Etc...

My use case is really trivial, I just capture region and paste result into mail or messenger window.
Actually Greenshot already has similar function called "Quick preferences". It even includes 'Effects' sub-menu, which sounds logical for this request, but we can review 'Capture mousepointer' option's use case.
What when the user **doesn't** want the effect? Then...he just unchecks this item in context menu, that's all

**Comment by gustavotrott** [comment]
I would like to see it as well.

If the user choose to open the image in editor, maybe this new option could be like an "automatic click" in 'Drop shadow'.
In this way, the effect would be applied after the image be opened in the editor and it would be possible to Undo.

![image](https://user-images.githubusercontent.com/5660191/119702748-2eec2b00-be2c-11eb-8008-d0b12f7129c3.png)

Not sure if this is the best approach.. just an idea.

**Comment by jmcahren** [comment]
Yep, this would be very useful.  I do a lot of technical writing and documentation, I take a lot of screenshots, and **_always_** apply the drop shadow.  

I think one way would be to have a 'default effect' setting, where you could enable/disable some common effects.
